README.md

```diff
@@ -1,10 +1,19 @@
-The leaderboard website displays scores submitted by different players.
-
-Screenshot (222)
+# Leaderboard
 
 The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
-Built With
+## Project Design:
+
+![screenshot](Screenshot.png)
+
+## Live Demo
+
+[Live Demo Link](http://localhost:8089/)
+
```

Comment:
- [ ] The live link is not working. Please update so that we can run your project directly. Currently, you have put the localhost link. If you don't have the live link then please remove the live link section so that the recruiter doesn't get confused.